_studio/shared/umc/core/umc/include/umc_va_base.h

```diff
@@ -286,6 +288,8 @@ class VideoAccelerator
     VideoAccelerationProfile    m_Profile;          // entry point
     VideoAccelerationPlatform   m_Platform;         // DXVA, LinuxVA, etc
     eMFXHWType                  m_HWPlatform;
+    uint32_t                    m_MaxContextPriority;
+    VideoCORE*                  m_core;
```

Comment:
Why do you need VideoCORE here? It introduces cross dependencies - you already have Core at component's level and can  send it as a parameter

Reply:
ok, I will change it and resubmit later.

Reply:
Done. Thanks.